brigade/templates/organize/index.html

```diff
@@ -3,19 +3,98 @@
 
 {% block content %}
 <section>
-  <div class="layout-semibreve">
 
-    <h2>Organizing a Brigade? We've got you covered.</h2>
+  <div class="layout-semibreve">
 
+    <h1 class="title">Resources</h1>
+    <p class="subtitle">Organizing a Brigade? We've got you covered.</p>
     <p>Over the past four years, we’ve learned a lot about what works when organizing a Brigade. We’re excited to help you build a sustainable, civic hacking group in your community.</p>
-
     <p>Here are some resources to help you get started:</p>
-    <p><a href="checklist/">Brigade Organizer's Checklist</a>: Learn how Code for America supports civic hack nights and Official Brigades.</p>
-    <p><a href="playbook/">Brigade Organizer’s Playbook</a>: Deep dive into three areas key to running a sustainable Brigade.</p>
-    <p><a href="{{ url_for('.free_software_index') }}">Free Software Available to Brigades</a>: Learn about the software that vendors have donated for Brigades to use.</p>
 
-    <p><a href="https://cfa.typeform.com/to/uxPQH6" class="button button-l" target="_blank"  title="New-to-organizing form">New to organizing?</a> Tell us a little bit about yourself so we can help you get started.</p>
+    <div class="card-row">
+
+      <a href="https://docs.google.com/document/d/19bN5RWK5nQTpz0mHUViHrzHiommBUAMSztwNRzUcxYo/" class="card card--horizontal" target="_blank">
+        <div class="card__aside">
+          <div class="card__icon"><img src="{{ asset_url_for('images/icon_playbook.png') }}"></div>
```

Comment:
No font-awesome? :(

Reply:
Couldn't find a Font Awesome icon that represented 'playbook' properly :(
One option is a clipboard: https://fontawesome.com/icons/clipboard?style=regular
What do you think? Is that better than the current image?